sdk/state/integration/helpers_test.go

```diff
@@ -17,7 +17,7 @@ import (
 
 // functions to be used in the state_test integration tests
 
-func initAccounts(t *testing.T) (initiator Participant, responder Participant) {
+func initAccounts(t *testing.T, client horizonclient.ClientInterface, asset txnbuild.Asset, assetLimit string, distributorKP *keypair.Full) (initiator Participant, responder Participant) {
```

Comment:
some refactoring in `initAccounts` so that we can create accounts with a non-native asset

Reply:
fyi this context is super helpful, thanks! 💯